downstream/modules/platform/proc-create-mappers.adoc

```diff
@@ -0,0 +1,30 @@
+[id="configure-mappers"]
+
+= Create mappers on the {PlatformNameShort} client
+
+[role=_abstract]
+The final step to take is to create the mappers on the {PlatformNameShort} {RHSSO} client. This defines the information coming from {RHSSO} and maps it to your {PlatformNameShort} users.
```

Comment:
The mapping is just in RHSSO, so this should read:
The final step to take is to create the mappers on the {RHSSO} client.